paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -108,14 +104,13 @@ internal sealed class PaymentSelection : Parcelable {
 
         override fun mandateText(
             merchantName: String,
-            isSaveForFutureUseSelected: Boolean,
             isSetupFlow: Boolean,
         ): ResolvableString? {
             return when (paymentMethod.type) {
                 USBankAccount -> {
                     USBankAccountTextBuilder.getContinueMandateText(
                         merchantName = merchantName,
-                        isSaveForFutureUseSelected = isSaveForFutureUseSelected,
+                        isSaveForFutureUseSelected = false,
```

Comment:
This is the only place this param `isSaveForFutureUseSelected` was used. But it's a saved payment method. So you can't ever have SFU set.